### PR TITLE
remove remaining collar voltage code from core library

### PR DESCRIPTION
**Pull request by ericgitonga**

### CODE REVIEW

File: ecoscope/plotting/__init__.py [hunk 1]
```diff
@@ -12,7 +11,6 @@
 __all__ = [
     "EcoPlotData",
     "add_seasons",
-    "collar_event_timeline",
```
Comment (walljcg) [inline ecoscope/plotting/__init__.py:15]: is this moved to a module now?

File: ecoscope/plotting/plot.py [hunk 2]
```diff
@@ -158,42 +150,6 @@ def add_seasons(fig, season_df):
     return fig
 
 
-def collar_event_timeline(relocations, collar_events):
```
Comment (walljcg) [inline ecoscope/plotting/plot.py:161]: where has this function moved to?

**Review by walljcg — CHANGES_REQUESTED** [review]

File: ecoscope/plotting/__init__.py (hunk 1 above)
Comment (ericgitonga) [reply ecoscope/plotting/__init__.py:15]: I did not include it, but can do so. Question, when is it used? I haven't seen an example where it has been called.

File: ecoscope/plotting/plot.py (hunk 2 above)
Comment (ericgitonga) [reply ecoscope/plotting/plot.py:161]: I have moved the function to the Collared Subject Report notebook, in the collar event cell. I have tested it to make sure it runs just fine.

**Pull request merged**